sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -624,10 +624,9 @@ def __init__(self, **kwargs):
     def _from_generated(cls, sentence):
         return cls(
             sentiment=sentence.sentiment,
-            confidence_scores=SentimentConfidenceScores._from_generated(sentence.sentence_scores),  # pylint: disable=protected-access
+            confidence_scores=SentimentConfidenceScores._from_generated(sentence.confidence_scores),  # pylint: disable=protected-access
             grapheme_offset=sentence.offset,
-            grapheme_length=sentence.length,
-            warnings=sentence.warnings,
```

Comment:
Remove docstring, instance var, repr, for warnings here, as well. Also doesn't `SentenceSentiment` now return `text`?